spot_rl_experiments/spot_rl/envs/base_env.py

```diff
@@ -769,6 +781,34 @@ def get_in_gripper_tf(self):
 
         return gripper_T_base
 
+    def get_gripper_position_in_base_frame_hab(self):
+        """
+        Returns the gripper position in the base frame as (x,z,-y) as per Habitat convention
+
+        Returns:
+            ee_pos_hab: Gripper position as an np.array([x,z,-y])
+        """
+
+        # Get end effector position (in base frame as per Habitat convention)
+        ee_pos_hab = np.array(self.get_in_gripper_tf().translation)
+        return ee_pos_hab
+
+    def get_gripper_position_in_base_frame_spot(self):
+        """
+        Returns the gripper position in the base frame as (x,y,z)
+
+        Returns:
+            ee_pos_spot: Gripper position as an np.array([x,y,z])
+        """
+        # Get end effector position (in base frame as per Hab convention)
+        ee_pos_hab = self.get_gripper_position_in_base_frame_hab()
+
+        # Convert ee_positoin (in base frame) to Spot convention .. (x,z,-y) -> (x,-y,z)
+        ee_pos_spot = ee_pos_hab[np.array([0, 2, 1])]
+        # Correct the sign on "y" .. (x,-y,z) -> (x,y,z)
+        ee_pos_spot[1] *= -1
+        return ee_pos_spot
```

Comment:
There are 2 conventions for reference frames in spot-sim2real - spot & habitat.

This adds getters for gripper position in base frame for both these conventions